src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
```

Comment:
> Whether the user is listening to the scratched signal in headphones, booth, or master output is irrelevant.

Yes, and in no case an artificial input delay is helpful. If you delay the headphones to match the master distance, you remove the double base, beatmatching becomes harder, because of the extra delay. If you cannot afford this extra latency, you can ditch the master distance compensation and isolate the master sound from the users ears.